src/special/commitizen.js

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
+
+export default function parseCommitizen(content, filePath, deps, rootDir) {
+  const packageJsonPath = path.resolve(rootDir, 'package.json');
+  const resolvedFilePath = path.resolve(filePath);
+
+  if (resolvedFilePath === packageJsonPath) {
```

Comment:
Oh, you are right. That keep this current behavior. (Another special may has this kind of issue, however...)